//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// acdef_NO_VP.h
//
// Code generation for function 'acdef_NO_VP'
//

#ifndef ACDEF_NO_VP_H
#define ACDEF_NO_VP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdef_NO_VP(double P_init, double V_init, double A_init, double P_wayp,
                 double V_wayp, double V_max, double A_min, double J_max,
                 double J_min, creal_T t[14]);

#endif
// End of code generation (acdef_NO_VP.h)
